core/modules/node/node.module

```diff
@@ -856,15 +856,15 @@ function node_invoke($node, $hook, $a2 = NULL, $a3 = NULL, $a4 = NULL) {
  * from the database. Nodes are loaded into memory and will not require database
  * access if loaded again during the same page request.
  *
- * @param $nids
- *   (optional) An array of node IDs.
- * @param $conditions
+ * @param array|false $nids
```

Comment:
Core seems to use `array|FALSE` instead of `array|false`. I couldn't fine any instance of the second.

Reply:
[Indicating data types in documentation](https://docs.backdropcms.org/doc-standards#types) in the coding standards says it is `false` not `FALSE`. I think that is the same type-hinting used in PHP documentation.

Reply:
Got it.

Reply:
I take that new code should follow what reported in the coding standards. Is it preferable to keep the code consistent even if it does not follow the coding standards? (I am asking just to understand what to do in future PRs.)

Reply:
@kiamlaluno Yep, what @argiepiano said. Always follow the coding standards for new/updated code. Then there'll be less issues to fix later.